content/protocol-adapters/index.textile

```diff
@@ -0,0 +1,44 @@
+
+The advantage of protocol adapters, however, is that they require fewer resources in terms of memory, and network overhead, so can work in smaller footprint devices, or on a platform where an Ably Client Library SDK is not available, such as on a Arduino-based IoT wearable. In this latter case, for example, MQTT clients are readily available for Arduino-based devices, and these can be used to connect to Ably, via the Ably MQTT protocol adapter.
+
+There are two protocol adapters covered in this section of the documentation:
+
+1. MQTT
+2. SSE
+
+Each of these will be described briefly in the following sections.
+
+h2(#mqtt). MQTT
+
+"MQTT":https://mqtt.org/ is an OASIS standard messaging protocol for the Internet of Things (IoT). It is designed as an extremely lightweight publish/subscribe messaging transport that is ideal for connecting remote devices with a small code footprint and minimal network bandwidth.
+
+The Ably MQTT protocol adapter provides an interface between an MQTT client and Ably. Being a protocol designed for small systems and IoT devices, MQTT has a low overhead in terms of system resource requirements. MQTT libraries are widely available, and support a range of languages such as C/C++, Python, and Rust. Because of this, client platforms and programming languages where an Ably Client Library SDK is not available, can still access the Ably realtime network through the MQTT client and the Ably MQTT protocol adapter.
+
+MQTT clients can subscribe and publish to Ably channels. Any client that supports MQTT can therefore leverage Ably's realtime network, with some constraints. 
+
+For more information on Ably's MQTT protocol adapter see the "MQTT documentation":/mqtt.
+
+h2(#sse). SSE
+
+Server-Sent Events (SSE) is a functionality built into more recent versions of browsers. Because "SSE support":https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events is built-in, it does not require the loading of any additional libraries, such as is the case if you were to use the Ably JavaScript Client Library SDK. You will have to handle certain operations in your client, for example, token renewal will need to be managed by your client code.
+
+The Ably SSE protocol adapter allows the browser to subscribe to a realtime stream of events from an Ably channel. Publishing to a channel is not supported by SSE.
+
+In addition to the SSE endpoint, Ably also provides a raw HTTP streaming endpoint. 
```

Comment:
Worth explaining in a single line what the difference is between the two, the readers will have no context at this point.

Reply:
Done!